tarea/task-2-class6-salary-form-bootstrap/task-2-class-10-salary-scripts.js

```diff
@@ -0,0 +1,155 @@
+const $body = document.querySelector("body");
+const $buttonAdd = document.querySelector("#add-input-button");
+const $buttonRemove = document.querySelector("#remove-input-button");
+const $salaryFormContent = document.querySelector("#salary-form-content");
+const $salaryForm = document.querySelector("#salary-form");
+const $buttonCalculateSalary = document.querySelector("#calculate-salary-button");
+const $resultContent = document.querySelector("#result-values-salary");
+
+const $salaryFormTitle = createTitle("h4", "salary-form-title", "Enter the annual salary of each family member, only one salary per input");
+$salaryFormContent.insertBefore($salaryFormTitle, $salaryForm);
+
+$buttonAdd.onclick = () => {
+    $salaryFormContent.style.display = "block";
+    $salaryFormTitle.style.display = "block";
+
+    const $salaryInputsStructure = createInputStructure();
+    $salaryForm.insertBefore($salaryInputsStructure, $buttonCalculateSalary);
+    return false;
+}
+
+$buttonRemove.onclick = () => {
+    const $labels = document.querySelectorAll(".salary-label");
+    const $inputs = document.querySelectorAll(".salary-input");
+
+    $labels[$labels.length - 1].remove();
+    $inputs[$inputs.length - 1].remove();
+    $resultContent.style.display = "none";
+
+    cleanUpInput($inputs);
+    return false;
+}
+
+$buttonCalculateSalary.onclick = () => {
+    const salariesFromHtml = document.querySelectorAll(".salary-input");
+    const salaries = getNumbers(salariesFromHtml);
+
+    const higherSalary = getHigher(salaries);
+    const smallerSalary = getSmaller(salaries);
+    const averageSalary = getAverage(salaries);
+    const averageMonthlySalary = getMonthlySalary(averageSalary);
+
+    $resultContent.style.display = "block";
+    const $higherSalaryText = document.querySelector("#higher-salary");
+    const $smallerSalaryText = document.querySelector("#smaller-salary");
+    const $averageSalaryText = document.querySelector("#average-salary");
+    const $averageMonthlySalaryText = document.querySelector("#average-monthly-salary");
+    $higherSalaryText.innerText = `The higher salary is: ${higherSalary}`;
+    $smallerSalaryText.innerText = `The smaller salary is: ${smallerSalary}`;
+    $averageSalaryText.innerText = `The average annual salary is: ${averageSalary}`;
+    $averageMonthlySalaryText.innerText = `The average monthly salary is: ${averageMonthlySalary}`;
+    return false;
+}
+
+function createTitle(type, className, message){
+    const $title = document.createElement(type);
+    $title.classList.add = className;
+    $title.innerText = message;
+    return $title;
+}
+
+function createInputStructure(){
+    const $row = createRow();
+    const $col = createCol(2);
```

Comment:
Why 2? What's 2? What's the reason to pass 2 as an argument? What's the meaning of that 2?